From 776bc883ab3d6f4d88343bec94c6faff373d5dd7 Mon Sep 17 00:00:00 2001
From: Arnaud Ferraris <arnaud.ferraris@gmail.com>
Date: Wed, 1 Dec 2021 20:48:51 +0100
Subject: [PATCH 309/515] arm64: dts: rk3399-pinephone-pro: Fix leds

Existing LEDs use different names and functions than what
we're used to, change them all to the same names we use on the OG
PinePhone.

Signed-off-by: Arnaud Ferraris <arnaud.ferraris@gmail.com>
---
 .../dts/rockchip/rk3399-pinephone-pro.dts     | 20 ++++++-------------
 1 file changed, 6 insertions(+), 14 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index fc16d7de4459..c89bc676b12c 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -127,30 +127,22 @@ leds {
 		pinctrl-names = "default";
 		pinctrl-0 = <&red_led_pin &green_led_pin &blue_led_pin>;
 
-		led-standby {
+		led-red {
 			color = <LED_COLOR_ID_RED>;
-			default-state = "off";
-			function = LED_FUNCTION_STANDBY;
+			function = LED_FUNCTION_INDICATOR;
 			gpios = <&gpio4 RK_PD2 GPIO_ACTIVE_HIGH>;
-			label = "red:standby";
-			panic-indicator;
-			retain-state-suspended;
 		};
 
-		led-pwr {
+		led-green {
 			color = <LED_COLOR_ID_GREEN>;
-			default-state = "on";
-			function = LED_FUNCTION_POWER;
+			function = LED_FUNCTION_INDICATOR;
 			gpios = <&gpio4 RK_PD5 GPIO_ACTIVE_HIGH>;
-			label = "green:disk-activity";
 		};
 
-		blue-charging {
+		led-blue {
 			color = <LED_COLOR_ID_BLUE>;
-			default-state = "off";
-			function = LED_FUNCTION_CHARGING;
+			function = LED_FUNCTION_INDICATOR;
 			gpios = <&gpio4 RK_PD6 GPIO_ACTIVE_HIGH>;
-			label = "blue:charging";
 		};
 	};
 
-- 
2.35.3

